README.md

```diff
@@ -34,18 +34,16 @@ dnf install pre-commit
 pre-commit run -a
 
 # test
-python3 -m pip install -e .[dev]
-pytest python/tests/unit
-pytest python/tests/integration
-```
+python3 -m pip install -e .
+python -m unittest discover tests
```

Comment:
Can you make these two lines of code reference the same Python interpreter? That is, can both lines either reference `python3` or `python`?